outputs.tf

```diff
@@ -0,0 +1,7 @@
+output "webapp_url" { 
+    value = "https://${azurerm_app_service.main.default_site_hostname}" 
+}
+
+output "cd_webhook" { 
```

Comment:
Worth noting that we don't want this output to be shared, so we should mark it [sensitive](https://www.terraform.io/docs/language/values/outputs.html#sensitive-suppressing-values-in-cli-output) here to avoid it leaking during our pipelines, otherwise we expose the opportunity for anyone to trigger an application deployment